README.md

```diff
@@ -3,7 +3,7 @@ by [Alphacamp](http://sg.alphacamp.co)
 
 ### Squirrel ERD
 
-- [View ERD](#)
+- [View ERD](https://s3-ap-northeast-1.amazonaws.com/assets.alphacamp.co/attachments/uploads/000/000/147/original/squirrel_v3.jpg?1510576435)
```

Comment:
Great stuff! Since it automatically downloads when you click it, could you change the link text from `View ERD` to `Download ERD`? :)